eng/targets/GeneratePkgDef.targets

```diff
@@ -293,12 +302,10 @@
     <Error Text="GeneratePkgDefFile is true but the project did not produce any entries (PkgDef* items) to be written to pkgdef file"
            Condition="'@(_PkgDefLines)' == ''"/>
 
-    <!-- Write final pkgdef content. If the CTO file was changed, touch the pkgdef file to cause a re-merge (see VSSDK targets). -->
     <WriteLinesToFile File="$(_GeneratePkgDefOutputFile)"
                       Lines="@(_PkgDefLines)"
                       Overwrite="true"
-                      Encoding="UTF-8"
-                      WriteOnlyWhenDifferent="!$([MSBuild]::ValueOrDefault('$(CTOFileHasChanged)', 'false'))" />
```

Comment:
Having a WriteOnlyWhenDifferent can break other up to date checking -- because we already have the inputs and outputs specified for this target, the timestamp being updated on an input pkgdef can cause this to get 'stuck' and never actually update the output file, continually resulting things not-up-to-date.